tests/admin_autocomplete/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
```

Comment:
Why did you create a new test application apart from all the other tests for this PR?

Reply:
Hm... I don't know. I didn't really see a structure in the whole test suite, nor did I find any documentation about the test suite structure.
If you have any suggestions I am more than happy to move things around.